cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1175,7 +1175,7 @@ features:
         b1: Intuitive Web UI for building, training, testing and evaluating Machine Learning models.
         b2: Seamless, integrated experience from data preparation and cleansing to model building and deployment.
         b3: Horizontal scalability over your Big Data environment.
-        b4: Out of the box support for common machine learning libraries like SparkML, DLF4J and H2O, along with support for deploying custom algorithms and libraries.
+        b4: Out of the box support for common machine learning libraries like SparkML.
```

Comment:
Should also remove the corresponding use in the js file. `/Users/ajainarayanan/Projects/cdap/cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js` ?

Reply:
I haven't removed this line. Only updated it.

Reply:
I haven't removed this line. Only updated it.

Reply:
oops sorry. 👍